### PR TITLE
gui-apps/kanshi: autorandr for wayland clients

### PR DESCRIPTION
**Pull request by epsilon-0**
Closes: https://bugs.gentoo.org/723746

kanshi allows you to define output profiles that
are automatically enabled and disabled on hotplug.
For instance, this can be used to turn a laptop's
internal screen off when docked.

Package-Manager: Portage-2.3.99, Repoman-2.3.22
Signed-off-by: Aisha Tammy <gentoo@aisha.cc>

---

@flappyports whenever you want, again, no rush. thanks a lot!

### CODE REVIEW

File: gui-apps/kanshi/kanshi-9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+SLOT="0"
+IUSE="doc"
+
+RDEPEND="
+	dev-libs/wayland
+"
+BDEPEND="
+	${RDEPEND}
+	doc? (
+		app-text/scdoc
```
Comment (flappyports) [inline gui-apps/kanshi/kanshi-9999.ebuild:29]: This should depend on the live ebuild for app-text/scdoc. See gui-wm/sway for an example.

File: gui-apps/kanshi/metadata.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        kanshi allows you to define output profiles that
+        are automatically enabled and disabled on hotplug.
+        For instance, this can be used to turn a laptop's
+        internal screen off when docked.
+    </longdescription>
+    <maintainer type="person">
+        <email>gentoo@aisha.cc</email>
+        <name>Aisha Tammy</name>
+    </maintainer>
+    <maintainer type="project">
```
Comment (flappyports) [inline gui-apps/kanshi/metadata.xml:14]: Please remove from m-n. I will be the Gentoo maintainer and you can be the main contact.

File: gui-apps/kanshi/kanshi-1.1.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+	doc? (
+		app-text/scdoc
+	)
+	virtual/pkgconfig
+	dev-libs/wayland-protocols
+"
+
+src_configure() {
+	local emesonargs=(
+		$(meson_feature doc man-pages)
```
Comment (flappyports) [inline gui-apps/kanshi/kanshi-1.1.0.ebuild:37]: man-pages should be behind a USE flag.

Comment (flappyports) [reply gui-apps/kanshi/kanshi-1.1.0.ebuild:37]: Also, doc should be behind a USE flag as well.

Comment (Alessandro-Barbieri) [reply gui-apps/kanshi/kanshi-1.1.0.ebuild:37]: nope
https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0301

Comment (epsilon-0) [reply gui-apps/kanshi/kanshi-1.1.0.ebuild:37]: Ah, I'll make it always on then.

Comment (flappyports) [reply gui-apps/kanshi/kanshi-1.1.0.ebuild:37]: We still want it to be configurable... Especially for folks working on things like embedded etc who strip lots of things.

File: gui-apps/kanshi/kanshi-9999.ebuild (hunk 1 above)
Comment (epsilon-0) [reply gui-apps/kanshi/kanshi-9999.ebuild]: I've added the sway version.

File: gui-apps/kanshi/metadata.xml (hunk 2 above)
Comment (epsilon-0) [reply gui-apps/kanshi/metadata.xml]: done.

File: gui-apps/kanshi/kanshi-1.1.0.ebuild (hunk 3 above)
Comment (epsilon-0) [reply gui-apps/kanshi/kanshi-1.1.0.ebuild]: done

**Comment by epsilon-0** [comment]
thanks a lot for your work!!

**Pull request closed without merging**